### PR TITLE
change: split training and serving logic

### PR DESCRIPTION
**Pull request by mvsusp**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=1aebaf6f-9f7c-457b-aa13-49ff06fb0f71%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=de89dc83-2f0c-4fb2-b5b6-de0ab4dcd6a1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=97d3a9a8-8eab-4607-b778-acf818485e58%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=838dc371-81d2-4bda-9159-77e7b5c6b834%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=6e58ee0d-35e6-4fa3-b4e6-1324ad8fa93d%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=2c63c024-6781-42e1-af1f-e3aa5aba7ffb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=89717e38-3f02-44a1-80f8-331617f1f924%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=293be02d-d493-495f-9153-44cdd942cc40%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=9a7a8f81-b10f-4010-9718-5c585391e8f2%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: buildspec.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 0.2
 env:
   variables:
     FRAMEWORK_VERSION: '1.1.0'
-    CPU_PY_VERSION: '2'
+    CPU_PY_VERSION: '3'
```
Comment (nskool) [inline buildspec.yml]: I'm assuming we'll re-add py2 support back in a later CR.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://vvaldn0il2.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=fd25d2f0-b620-4eb1-856e-7f68f526708a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by axelborja** [comment]
In this PR, the serving seems to disappear, can we find it somewhere else ?

**Comment by manubhat90** [comment]
> In this PR, the serving seems to disappear, can we find it somewhere else ?

https://github.com/aws/sagemaker-pytorch-serving-container

**Pull request merged**